sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1837,6 +1841,35 @@ export class ApplicationService extends RecordDataModelService<Application> {
     });
   }
 
+  /**
+   * Get history of application versions for an application
+   * where the current application is excluded.
+   * @param applicationId application id.
+   * @returns assessment history list.
+   */
+  async applicationVersionHistory(
+    applicationId: number,
+  ): Promise<Application[]> {
+    const application = await this.repo.findOne({
+      select: { applicationNumber: true },
+      where: { id: applicationId },
+    });
+    if (!application) {
+      return null;
+    }
```

Comment:
To check if the application exists should not be the responsibility of this method. 

Ref: https://github.com/bcgov/SIMS/pull/4213/files#r1907713751